include/api.php

```diff
@@ -779,6 +780,37 @@ function api_get_user(App $a, $contact_id = null)
 		'network' => $uinfo[0]['network'],
 	);
 
+	// If this is a local user and it uses Frio, we can get its color preferences.
+	if ($ret['self']) {
+		$theme_info = dba::select('user', ['theme'], ['uid' => $ret['uid']], ['limit' => 1]);
+		if ($theme_info['theme'] === 'frio') {
+			$schema = PConfig::get($ret['uid'], 'frio', 'schema');
+			if (($schema) && ($schema != '---')) {
```

Comment:
Please remove extra parentheses in this condition.

Reply:
I, personally, would like to keep the one around ```($schema != '---')``` but remove the one around ```($schema)```. Although not needed, it increases readability.